src/plots/gl3d/layout/axis_defaults.js

```diff
@@ -51,10 +57,13 @@ module.exports = function supplyLayoutDefaults(layoutIn, layoutOut, options) {
         if(coerce('showspikes')) {
             coerce('spikesides');
             coerce('spikethickness');
-            coerce('spikecolor');
+            // TODO: this makes the default spikes #444 rather than
+            // #000, is that OK?
+            coerce('spikecolor', containerOut.color);
```

Comment:
Good. I'll take that note out then. If it wasn't clear, the reason for this change is so `spikecolor` will follow `axis.color` by default, and `axis.color` defaults to `'#444'`